setup.rb

```diff
@@ -11,8 +11,7 @@
   ENV.delete 'RUBYOPT'
 
   require 'rbconfig'
-  cmd = [RbConfig.ruby, 'setup.rb', *ARGV].compact
-  cmd[1,0] = "--disable-gems"
+  cmd = [RbConfig.ruby, '--disable-gems', 'setup.rb', *ARGV].compact
```

Comment:
:thinking: Do you have any idea why `compact` is needed at all? `RbConfig.ruby` will never be `nil` and even if it would be `nil`, it will fail later. `ARGV` is an array (but even it would be `nil`, deconstruction will handle it properly).

Reply:
It seems completely unnecessary indeed, I can remove that too since we're at it.

Reply:
Done!